toffy/normalize_test.py

```diff
@@ -0,0 +1,196 @@
+import json
+import os
+import pytest
+
+import numpy as np
+import pandas as pd
+import tempfile
+
+from pytest_cases import parametrize_with_cases
+
+from ark.utils import test_utils, load_utils
+from toffy import normalize
+import toffy.normalize_test_cases as test_cases
+
+parametrize = pytest.mark.parametrize
+
+
+# TODO: move to toolbox repo once created
+def _make_blank_file(folder, name):
+    with open(os.path.join(folder, name), 'w'):
+        pass
+
+
+@parametrize('obj_func_name, num_params', [('poly_2', 3), ('poly_3', 4), ('poly_4', 5),
+                                           ('poly_5', 6), ('log', 2), ('exp', 4)])
+def test_create_objective_function(obj_func_name, num_params):
+
+    obj_func = normalize.create_objective_function(obj_func_name)
+
+    # number of weights + 1 for x
+    inputs = [1] * (num_params + 1)
+
+    _ = obj_func(*inputs)
+
+
+@parametrize('plot_fit', [True, False])
+@parametrize('obj_func', ['poly_2', 'poly_3', 'poly_4', 'poly_5', 'log', 'exp'])
+def test_fit_calibration_curve(plot_fit, obj_func):
+    x_vals = np.random.rand(15)
+    y_vals = np.random.rand(15)
+    _ = normalize.fit_calibration_curve(x_vals, y_vals, obj_func, plot_fit)
+
+
+@parametrize('obj_func, num_params', [('poly_2', 3), ('poly_3', 4), ('poly_4', 5),
```

Comment:
this is one place where parametrizing a fixture, instead of the test case, can reduce the number of times these arguments are manually specified. not necessary to change imo, but something we should be aware of for future testing convention.